client/static/css/js/index.js

```diff
@@ -0,0 +1,69 @@
+const form = document.querySelector("form");
+
+form.addEventListener("submit", async (e) => {
+    e.preventDefault();
+
+    const output = {
+        title: e.target.title.value,
+        pseudonym: e.target.pseudonym.value,
+        body: e.target.body.value,
+    };
+
+    // Here We're checking none of the inputs are falsy, if they are we aren't sending them to the server because they're required in our database.
+    for (const key in output) {
+        if (!output[key]) {
+            return
+        }
+    }
+
+    const options = {
+        method: 'POST',
+        headers: {
+            'Content-Type': 'application/json'
+        },
+        body: JSON.stringify(output)
+    }
+    
+
+    // TODO This should return the id of the newly recrated object.
+    const response = await fetch("http://localhost:3000/posts/new", options);
+
```

Comment:
We were attempting something similar, In theory if you can access the id from the object array thats produced on input, you could append it onto a string that redirects to the next html page. And then on the next html page, you can access that fetched data using a show path from your route.